docs/getting-started-on-snowplow-open-source/quick-start-aws/index.md

```diff
@@ -88,95 +96,78 @@ terraform plan
-|--------------------|------------------------------------------------|
-| account            | The account name.                              |
-| username           | A snowflake user to perform resource creation. |
-| region             | Region for the snowflake deployment.           |
-| role               | Needs to be ACCOUNTADMIN or similar.           |
-| private_key_path | Path the private key.                          |
+* Snowflake: https://github.com/snowplow-devops/terraform-aws-snowflake-loader-ec2#usage
+* Databricks: https://github.com/snowplow-devops/terraform-aws-databricks-loader-ec2#usage
+* Redshift: https://github.com/snowplow-devops/terraform-aws-redshift-loader-ec2#usage
 
-#### Usage
+Take note of all of the inputs/outputs produced as you will need them to flesh out the `<destination>_*` variables in the `terraform.tfvars` file.
 
-1. Fill variables in [terraform.tfvars](https://github.com/snowplow/quickstart-examples/blob/main/terraform/aws/snowflake/terraform.tfvars) within the `aws/snowflake` folder. Snowflake connection details found in the [Prerequisites](#Prerequisites) section need to be assigned to respective variables in `terraform.tfvars`.
-2. Run `terraform init`
-3. Run `terraform apply`
-
-#### Output
-
-Snowflake Terraform module will output the name of the created resources. Full list can be found [here](https://github.com/snowplow/quickstart-examples/blob/snowflake-loader/terraform/aws/snowflake/outputs.tf).
-
-These output values need to be passed to `aws/pipeline` modules as a variable when Snowflake is selected as pipeline's destination.
-
-**Step 3: Run the terraform script to set up your Pipeline stack**
+**Step 2: Run the terraform script to set up your Pipeline stack**
 
 You can now use terraform to create your Pipeline stack. You will be asked to select a region, you can find more information about [available regions here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html).
 
 ```bash
 terraform init
-terraform plan -var-file=<destination>.terraform.tfvars
-terraform apply -var-file=<destination>.terraform.tfvars
+terraform plan
+terraform apply
 ```
 
-This will output your `collector_dns_name`, `db_address`, `db_port` and `db_id`. Make a note of these, you'll need it when sending events and connecting to your database. If you have attached a custom ssl certificate and set up your own DNS records then you don't need your `collector_dns_name` as you will use your own DNS record to send events from the Snowplow trackers.
+This will output your `collector_dns_name`, `postgres_db_address`, `postgres_db_port` and `postgres_db_id`. Make a note of these, you'll need it when sending events and connecting to your database. If you have attached a custom ssl certificate and set up your own DNS records then you don't need your `collector_dns_name` as you will use your own DNS record to send events from the Snowplow trackers.
```

Comment:
The postgres ones only appear if you chose postgres, right? I would clarify that.

Reply:
They always appear annoyingly - they are just empty values.  But yes that should be clarified.